packages/foreman/foreman/foreman.provreq

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python2
+
+"""
+Automatic provides and requires generator for webpack bundles
+"""
+
+import json
+import os
+import sys
+
+FOREMAN_MANIFEST = '/var/lib/foreman/public/webpack/manifest.json'
+
+paths = [path.rstrip() for path in sys.stdin.readlines()]
+
+mode = os.path.splitext(sys.argv[0])[1].lstrip('.')
+
+def foreman_webpack(name, chunk):
+    filename, fileext = os.path.splitext(chunk)
+    fileext = fileext.lstrip('.')
+    filehash = filename.rsplit('-', 1)[1]
+    return 'foreman-webpack-{}-{}({})'.format(name, fileext, filehash)
+
+def parse_manifest(path, mode):
+    result = set()
```

Comment:
Using the same variable name as in the top scope is usually not a good idea. Unless you hide that in a `main()`